docs/docsite/rst/community/maintainers_guidelines.rst

```diff
@@ -52,6 +52,21 @@ Collection contributors and maintainers should also communicate through:
 
 See :ref:`communication` for more details on these communication channels.
 
+.. _wg_and_real_time_chat:
+
+Working Group and Real-time Chat Request
+----------------------------------------
+
+Here are some techniques for communicating more efficiently and interactively within your community:
```

Comment:
```suggestion
Establishing chat communication as a working group
-----------------------------------------------------------------------------

Working groups depend on efficient, real-time communication.
Project maintainers can use the following techniques to establish communication for working groups:
```

Reply:
How about this? The heading as-is doesn't seem to set the context quite right imo.

Reply:
i wasn't sure if chats is the only way they can communicate but as there's nothing else in communication guide Working Groups section (e.g. no mailing list, etc, only matrix/irc) I think it's fair to have `chat communication` in the title.
Should we put a blank line between lines 60 and 61 for readability?

Reply:
yeah maybe "chats" isn't quite right. `s/Establishing chat communication/Establishing communication`

I put each sentence on a separate line as a force of habit. ventilated prose ftw. I don't think a blank line is really necessary.

Reply:
wfm, committed, PTAL